src/interfaces/wallet.ts

```diff
@@ -15,12 +15,11 @@ export interface ITransaction {
   balance: number;
   timestamp: number;
   notes: string;
-  trxLogId: string;
+  trxLogId?: string;
```

Comment:
This is not ideal. We always want to have a log entry for all wallet transactions.

Reply:
On another thought, @nomadyus is right. please revert this code @Md-coder